routes.js

```diff
@@ -67,6 +70,11 @@ router.get('/event-group/:id', auth.authUser, (req, res) => eventGropController.
 router.patch('/event-group', auth.authAdmin, (req, res) => eventGropController.update(req, res))
 router.delete('/event-group/:id', auth.authAdmin, (req, res) => eventGropController.remove(req, res))
 
+// Notification routes
+router.post('/notification', auth.authAdmin, (req, res) => notificationController.create(req, res));
+router.get('/notification/:read', auth.authUser, (req, res) => notificationController.getAll(req, res));
+router.get('/notification/get-by-id/:id', auth.authUser, (req, res) => notificationController.getById(req, res));
```

Comment:
Path needs no 'get-by-id' part, '/notification/:id' is enough, I see why you did it, it collided with the get all method with the 'read' param, but in my opinition the read param should be sent via request body, It's more like a filtering param so I think that it shouldn't be in the params of the URL.

Reply:
Will be changed by days end. Didn't even think to send it through the body.

Reply:
The more I think about it, the more I hate it. Passing a single one(1) parameter to a GET request should not be done through the request's body, but at the same time keeping `get-by-id` is also fucking stupid. I'm not sure what to do since both ideas seem stupid af.

Reply:
Make a compromise and do the all route like '/notification/all/:read' and the get by id one like '/notifcation/:id'

Reply:
Makes the most sense in my opinion.